osu.Game/Screens/OnlinePlay/DrawableRoomPlaylistItem.cs

```diff
@@ -105,10 +107,18 @@ private void scheduleRefresh()
 
         private void refresh()
         {
+            var beatmapMetadata = Item.Beatmap.Value.Metadata ?? Item.Beatmap.Value.BeatmapSet?.Metadata;
```

Comment:
the null propagation on `Item.Beatmap.Value.BeatmapSet` not only causes an inspection to trip on line 116 (because the metadata is now possibly-null), it triggers another one on code that hasn't been changed here (line 131, explicit content check).

not sure what the situation here is with regard to nullability. `BeatmapInfo` does the same, fwiw:

https://github.com/ppy/osu/blob/1832f624e20a1b0c5ca9365e0c31abc29be1a148/osu.Game/Beatmaps/BeatmapInfo.cs#L146-L151

either way this needs to be fixed somehow.

side note is that maybe we want to remove usages of that `BeatmapInfo.ToString()` and have an equivalent `RomanisableString` that would be a drop-in replacement here (and spare the weird sprite text construction).